deny.toml

```diff
@@ -36,21 +40,26 @@ wildcards = "deny"
 highlight = "all"
 # Certain crates/versions that will be skipped when doing duplicate detection.
 skip = [
-    { name = "ahash", version = "0.4" },
-    { name = "cfg-if", version = "0.1" },             # https://github.com/rustwasm/console_error_panic_hook/pull/18
-    { name = "core-foundation", version = "0.6" },
-    { name = "core-foundation", version = "0.7" },
-    { name = "core-foundation-sys", version = "0.6" },
-    { name = "core-foundation-sys", version = "0.7" },
-    { name = "core-graphics", version = "0.19" },
-    { name = "fixedbitset", version = "0.2" },
-    { name = "hashbrown", version = "0.9" },
-    { name = "libm", version = "0.1" },
-    { name = "mach", version = "0.2" },
-    { name = "ndk", version = "0.3" },
-    { name = "ndk-glue", version = "0.3" },
-    { name = "proc-macro-crate", version = "0.1" },
-    { name = "stdweb", version = "0.1" },
+    { name = "cfg-if", version = "0.1" },               # from winit v0.26.0
+    { name = "core-foundation", version = "0.6" },      # from gilrs v0.8.1
+    { name = "core-foundation", version = "0.7" },      # from winit v0.26.0
+    { name = "core-foundation-sys", version = "0.6" },  # from gilrs v0.8.1
+    { name = "core-foundation-sys", version = "0.7" },  # from winit v0.26.0
+    { name = "core-graphics", version = "0.19" },       # from winit v0.26.0
+    { name = "darling", version = "0.10" },             # from rodio v0.14.0
+    { name = "darling_core", version = "0.10" },        # from rodio v0.14.0
+    { name = "darling_macro", version = "0.10" },       # from rodio v0.14.0
+    { name = "mach", version = "0.2" },                 # from gilrs v0.8.1
+    { name = "mio", version = "0.7" },                  # from notify v5.0.0-pre.11
+    { name = "ndk", version = "0.3" },                  # from rodio v0.14.0
```

Comment:
Why does rodio itself pull in two versions of `ndk`?

Reply:
rodio is using oboe which depends on ndk 0.4 and cpal which depends on ndk 0.3. I only noted the dependency from Bevy, not the complete dependency path